src/qml/qmlwaveformrenderer.h

```diff
@@ -19,8 +19,12 @@ class WaveformRenderBeat;
 namespace mixxx {
 namespace qml {
 
+typedef ::WaveformRendererAbstract::PositionSource WaveformRendererPositionSource;
```

Comment:
NIT. also add a comment why the alias exists.
```suggestion
using WaveformRendererPositionSource = ::WaveformRendererAbstract::PositionSource;
```